sycl/doc/GetStartedGuide.md

```diff
@@ -704,49 +704,38 @@ SYCL_BE=PI_CUDA ./simple-sycl-app-cuda.exe
 ```
 
 **NOTE**: DPC++/SYCL developers can specify SYCL device for execution using
-device selectors (e.g. `sycl::cpu_selector`, `sycl::gpu_selector`,
+device selectors (e.g. `sycl::cpu_selector_v`, `sycl::gpu_selector_v`,
 [Intel FPGA selector(s)](extensions/supported/sycl_ext_intel_fpga_device_selector.md)) as
 explained in following section [Code the program for a specific
 GPU](#code-the-program-for-a-specific-gpu).
 
 ### Code the program for a specific GPU
 
-To specify OpenCL device SYCL provides the abstract `sycl::device_selector`
-class which the can be used to define how the runtime should select the best
-device.
+To assist in finding a specific OpenCL device out of all that may be available, a "device selector" may be used. A "device selector" is a ranking function that will give an integer ranking value to all the
```

Comment:
Do we need OpenCL here?

Reply:
good catch. replaced it with "SYCL compatible" which is the language from the spec